app/assets/json/translations/locale-zh.json

```diff
@@ -346,7 +346,7 @@
             "draft_application_saved": {
                 "applications_are_due_by": "該物業的申請截止日期為%{deadline}。",
                 "complete_your_application": "完成您的申請",
-                "heres_link_to_complete": "請訪問此連結完成您在DAHLIA三藩市房屋網站暫存的{listing_name}申請。",
+                "heres_link_to_complete": "請訪問此連結完成您在DAHLIA三藩市房屋網站暫存的 %{listing_name}申請。",
```

Comment:
how come this one is `%{listing_name}`, not `{{listing_name}}`?

Reply:
it's because it's ruby instead of angular because it's from an email. they have a different placeholder syntax.